addons/project/__manifest__.py

```diff
@@ -80,9 +80,13 @@
             'project/static/src/scss/project_sharing_frontend.scss',
             'project/static/src/js/portal_rating.js',
         ],
+        'web.assets_unit_tests': [
+            'project/static/tests/mock_server/**/*',
+        ],
         'web.qunit_suite_tests': [
             'project/static/src/project_sharing/components/portal_file_input/portal_file_input.js',
             'project/static/tests/**/*.js',
```

Comment:
This should exclude tours as well I believe ("mistake" from even before this PR).

Reply:
I tried but I ran into dependencies issues. Should I investigate further or can we move forward with the test conversion ?